spec/factories/school.rb

```diff
@@ -10,6 +10,7 @@
     creator_id { SecureRandom.uuid }
     creator_agree_authority { true }
     creator_agree_terms_and_conditions { true }
+    creator_agree_to_ux_contact { true }
```

Comment:
I think this isn't needed - we just create the factories with the minimal amount of data present to be valid

Reply:
shell I delete this line completely or just change it to false?

Reply:
no no what I meant was we don't need this attribute at all in the factory - unless you're validating its presence somewhere?

Reply:
Understood 🫡